android/src/main/java/com/swmansion/rnscreens/CustomToolbar.kt

```diff
@@ -19,16 +21,22 @@ open class CustomToolbar(
             override fun doFrame(frameTimeNanos: Long) {
                 isLayoutEnqueued = false
                 measure(
-                    MeasureSpec.makeMeasureSpec(width, MeasureSpec.EXACTLY),
-                    MeasureSpec.makeMeasureSpec(height, MeasureSpec.EXACTLY),
+                    MeasureSpec.makeMeasureSpec(width, MeasureSpec.AT_MOST),
+                    MeasureSpec.makeMeasureSpec(height, MeasureSpec.AT_MOST),
```

Comment:
1. This is fine as long as it does not break any other behaviour. Please check header behaviours carefully, especially on newer Android API levels (34, 35).

2. Please add a comment here why this is done. 
3. Maybe we should select the measure spec depending on the API level being in use? So that we're sure we do not change the behaviour for newer API levels?

Reply:
@kkafar I believe this layoutCallback is not triggered in any other circumstances. It is called conditionally when API <= 29 and softInputMode is set to "adjustPan" in the modified `requestLayout` in the same file. Thus I don't think we need to add conditions/comment here.

Reply:
Yes, I think you're right. I wrote this code but I've forgotten about it.